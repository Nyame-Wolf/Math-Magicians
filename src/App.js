/* eslint-disable react/prefer-stateless-function */
import React from 'react';
import { Routes, Route } from 'react-router-dom';
import Calculator from './components/Calculator';
import Footer from './components/Footer';
import Navigation from './components/Navigation';
import Home from './pages/Home';
import Quotes from './pages/Quotes';

const App = () => (

  <div>
    <div className="heading">
      <h1>Math Magicians</h1>
      <Navigation />
    </div>
    <Routes>
      <Route path="/" element={<Home />} />
      <Route path="/calculator" element={<Calculator />} />
      <Route path="quotes" element={<Quotes />} />
    </Routes>
    {/* <Outlet /> */}
    <Footer />
  </div>
);

export default App;
