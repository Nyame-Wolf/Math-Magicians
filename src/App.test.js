import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import App from './App';

describe('App', () => {
  test('renders App component', () => {
    render(
      <BrowserRouter>
        <App />
      </BrowserRouter>,
    );

    // screen.debug();
  });
  test('App Snapshot', () => {
    const tree = render(
      <BrowserRouter>
        <App />
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
