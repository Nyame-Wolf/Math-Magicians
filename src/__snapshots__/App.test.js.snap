// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="heading"
        >
          <h1>
            Math Magicians
          </h1>
          <ul
            class="nav"
          >
            <a
              aria-current="page"
              class="link active"
              href="/"
            >
              Home
            </a>
            <a
              class="link link-b"
              href="/calculator"
            >
              Calculator
            </a>
            <a
              class="link"
              href="/quotes"
            >
              Quotes
            </a>
          </ul>
        </div>
        <p
          class="info"
        >
          Nisi cillum exercitation cillum proident occaecat. Exercitation nisi aliquip irure id nostrud aliqua. Dolor aliquip ut occaecat duis labore laborum labore exercitation. Quis et non consequat id cupidatat sunt ut. Reprehenderit cupidatat ipsum est nisi minim culpa irure. Cillum ullamco anim sit est eiusmod et et veniam. Ex mollit sit amet adipisicing do in ex mollit ad.
        </p>
        <p
          class="info"
        >
          Nisi cillum exercitation cillum proident occaecat. Exercitation nisi aliquip irure id nostrud aliqua. Dolor aliquip ut occaecat duis labore laborum labore exercitation. Quis et non consequat id cupidatat sunt ut. Reprehenderit cupidatat ipsum est nisi minim culpa irure. Cillum ullamco anim sit est eiusmod et et veniam. Ex mollit sit amet adipisicing do in ex mollit ad.
        </p>
        <footer
          class="footer"
        >
          <p>
            <a
              class="footer-link"
              href="https://www.linkedin.com/in/mumenya-nyamu-web-designer-data-enthusiast/"
            >
              ©  Mumenya Nyamu 2022
            </a>
          </p>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="heading"
      >
        <h1>
          Math Magicians
        </h1>
        <ul
          class="nav"
        >
          <a
            aria-current="page"
            class="link active"
            href="/"
          >
            Home
          </a>
          <a
            class="link link-b"
            href="/calculator"
          >
            Calculator
          </a>
          <a
            class="link"
            href="/quotes"
          >
            Quotes
          </a>
        </ul>
      </div>
      <p
        class="info"
      >
        Nisi cillum exercitation cillum proident occaecat. Exercitation nisi aliquip irure id nostrud aliqua. Dolor aliquip ut occaecat duis labore laborum labore exercitation. Quis et non consequat id cupidatat sunt ut. Reprehenderit cupidatat ipsum est nisi minim culpa irure. Cillum ullamco anim sit est eiusmod et et veniam. Ex mollit sit amet adipisicing do in ex mollit ad.
      </p>
      <p
        class="info"
      >
        Nisi cillum exercitation cillum proident occaecat. Exercitation nisi aliquip irure id nostrud aliqua. Dolor aliquip ut occaecat duis labore laborum labore exercitation. Quis et non consequat id cupidatat sunt ut. Reprehenderit cupidatat ipsum est nisi minim culpa irure. Cillum ullamco anim sit est eiusmod et et veniam. Ex mollit sit amet adipisicing do in ex mollit ad.
      </p>
      <footer
        class="footer"
      >
        <p>
          <a
            class="footer-link"
            href="https://www.linkedin.com/in/mumenya-nyamu-web-designer-data-enthusiast/"
          >
            ©  Mumenya Nyamu 2022
          </a>
        </p>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
